azure-functions-maven-plugin/src/it/http-trigger-with-3rd-lib/pom.xml

```diff
@@ -1,140 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why delete the project?

Reply:
Because this project is using shaded plugin to generate a fat jar. Since the service side has already support non-fat jar functions and fat-jar is not a recommended way to package Function project, so I decide to remove this.

Reply:
then it isn't the reason why integration tests fail?

Reply:
Yes, the removal is just a refine.